### PR TITLE
Update Google entry on Sponsors page

### PR DESCRIPTION
**Pull request by gentlecat**

### CODE REVIEW

File: metabrainz/templates/index/sponsors.html [hunk 1]
```diff
@@ -24,8 +24,7 @@ <h1 class="page-title">Sponsors</h1>
       <td><img src="{{ url_for('static', filename='img/logos/sponsors/google.svg') }}"></td>
       <td>
         <b><a href="https://www.google.com/">Google</a></b> has been an extremely supportive
-        sponsor. They've donated over <b>$240,000</b> and they employ our treasurer and
-        secretary, Carol Smith.
+        sponsor. They've donated over <b>$240,000</b>.
       </td>
```
Comment (mayhem) [inline metabrainz/templates/index/sponsors.html]: $320k so far. I need to actually enter some more entries for google in the donation table.

Comment (gentlecat) [inline metabrainz/templates/index/sponsors.html]: Updated.

**Comment by mayhem** [comment]
:panda_face: :speak_no_evil: :heavy_check_mark:

**Comment by gentlecat** [comment]
:scream:

**Pull request merged**